SCClassLibrary/Common/Control/Server.sc

```diff
@@ -348,23 +370,42 @@ Server {
 	}
 
 	initTree {
-		nodeAllocator = NodeIDAllocator(clientID, options.initialNodeID);
-		this.sendMsg("/g_new", 1, 0, 0);
+		this.newNodeAllocators;
+		this.sendMsg("/g_new", this.defaultGroup.nodeID, 0, 0);
 		tree.value(this);
 		ServerTree.run(this);
 	}
 
 	/* id allocators */
 
+	// private, called from server notify response with next free clientID
 	clientID_ { |val|
+		var failstr = "Server % couldn't set client id to: % - %. clientID is still %.";
```

Comment:
let's capitalize ID here for consistency

Reply:
ok